test/check-machines-create

```diff
@@ -296,23 +296,20 @@ class TestMachinesCreate(VirtualMachinesCase):
                                                                            storage_pool="NoStorage",
                                                                            connection="session"))
 
-        self.machine.execute("virsh net-destroy default && virsh net-undefine default")
+        self.machine.execute("set -e; virsh net-destroy default; virsh net-undefine default")
 
         # Set up the PXE server configuration files
         cmds = [
+            "set -e",
```

Comment:
Since `set -e` was already called, isn't this one unecessary?

Reply:
Where was it called? Each self.machine.execute() call runs a new shell. I really wish that API would call the shell with `-e` by default.. It's really always what you want. I have it on my list to do that and fix the fallout in our projects, but let's not block on that.

Reply:
Ahh, I forgot it's called with a new shell, nevermind then